### PR TITLE
added search bar to ArtifactList and data gets shown on EditArtifact

### PR DESCRIPTION
**Pull request by FahadUmair**

### CODE REVIEW

File: src/final/components/TimelineItem.js
Comment (chahilr) [inline src/final/components/TimelineItem.js:1]: Delete this

File: src/final/components/VerticalSlider.js
Comment (chahilr) [inline src/final/components/VerticalSlider.js:1]: Delete this

File: src/final/components/TimelineItem.js
Comment (FahadUmair) [reply src/final/components/TimelineItem.js:1]: done

File: src/final/components/VerticalSlider.js
Comment (FahadUmair) [reply src/final/components/VerticalSlider.js:1]: done

**Review by chahilr — APPROVED** [review]

**Pull request merged**